legion/engine/physics/halfedgeface.cpp

```diff
@@ -392,6 +459,26 @@ namespace legion::physics
 
         };
 
+        math::vec3 worldStart = transform * math::vec4(centroid, 1);
+        math::vec3 worldEnd = transform * math::vec4(centroid + normal * 0.1f, 1);
+
+        //debug::user_projectDrawLine(worldStart, worldEnd, debugColor, 3.0f, time, false);
+
+        forEachEdge(drawFunc);
+    }
+
+    void HalfEdgeFace::DEBUG_DirectionDrawFace(const math::mat4& transform, const math::color& debugColor, float time)
+    {
+        auto drawFunc = [&transform, debugColor, time](HalfEdgeEdge* edge)
+        {
+            edge->DEBUG_directionDrawEdge(transform, debugColor, time, 5.0f);
+        };
+
+        math::vec3 worldStart = transform * math::vec4(centroid, 1);
+        math::vec3 worldEnd = transform * math::vec4(centroid + normal * 0.1f, 1);
+
+        debug::user_projectDrawLine(worldStart, worldEnd, math::colors::green, 3.0f, time, false);
```

Comment:
`debug::user_projectDrawLine` may not always exist at compile time, use `debug::DrawLine` instead